MediaAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/Collection/Media/Medias.js

```diff
@@ -40,9 +40,15 @@ class Medias extends DataTableCollection
      */
     _getSyncUrl(method, options) {
         let urlParameter = options.urlParameter || {};
+        let apiContext = options.apiContext || {};
+
         switch (method) {
             case "read":
-                return Routing.generate('open_orchestra_api_media_list', urlParameter);
+                let route = 'open_orchestra_api_media_list_with_perimeter';
+                if (apiContext.withoutPerimeter) {
```

Comment:
why not use the string like other apiContext. If we want to add an other context, Without refactoring the view